python/ray/_private/ray_logging.py

```diff
@@ -234,6 +236,26 @@ def configure_log_file(out_file, err_file):
     )
 
 
+def init_worker_logs(worker_type: str):
+    assert ray._private.worker._global_node is not None
+    out_file, err_file = ray._private.worker._global_node.get_log_file_handles(
+        get_worker_log_file_name(worker_type)
+    )
+    configure_log_file(out_file, err_file)
+
+
+def reconfigure_worker_logs(worker_type: str, job_id: str):
```

Comment:
nit: add a log to this file? It is changing some state, would be helpful to have. Unless this runs in the worker itself, then it's kinda difficult..

Reply:
yeah this runs in worker